### ISSUE TITLE
Predicting Physical Object Properties from Video. (arXiv:2206.00930v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#19609]
Predicting Physical Object Properties from Video. (arXiv:2206.00930v1 [cs.CV])<br>
https://ift.tt/cNCgQZz<br>
<p>We present a novel approach to estimating physical properties of objects from video. Our approach consists of a physics engine and a correction estimator. Starting from the initial observed state, object behavior is simulated forward in time. Based on the simulated and observed behavior, the correction estimator then determines refined physical parameters for each object. The method can be iterated for increased precision. Our approach is generic, as it allows for the use of an arbitrary - not necessarily differentiable - physics engine and correction estimator. For the latter, we evaluate both gradient-free hyperparameter optimization and a deep convolutional neural network. We demonstrate faster and more robust convergence of the learned method in several simulated 2D scenarios focusing on bin situations.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
